petrel/auth.go

```diff
@@ -64,22 +64,30 @@ func newAuthServer(serverIP string, port int, vpnnet string) (*AuthServer, error
 	return a, err
 }
 
+func dumpString(s string) {
+	fmt.Println("dump start. len=", len(s))
+	for i := 0; i < len(s); i++ {
+		fmt.Printf("%x ", s[i])
```

Comment:
You can use `fmt.Printf("% x", s) instead of a loop to print the bytes with space between them.